cfme/tests/cloud/test_tenant.py

```diff
@@ -211,3 +212,23 @@ def test_dynamic_product_feature_for_tenant_quota(request, appliance, new_tenant
     # tenant2(child tenant).
     with user_[1]:
         check_permissions(appliance=appliance, assigned_tenant=child_tenant.name)
+
+
+@test_requirements.quota
+@pytest.mark.tier(2)
+def test_tenant_quota_input_validate(appliance):
+    """
+    Polarion:
+        assignee: ghubale
+        casecomponent: Quota
+        caseimportance: medium
+        initialEstimate: 1/8h
+    """
+    roottenant = appliance.collections.tenants.get_root_tenant()
+    fields = [('cpu', 2.5), ('storage', '1.x'), ('memory', '2.x'), ('vm', 1.5)]
+
+    for field in fields:
+        view = navigate_to(roottenant, 'ManageQuotas')
+        view.form.fill({'{}_cb'.format(field[0]): True, '{}_txt'.format(field[0]): field[1]})
```

Comment:
Please include inline comment unpacking what's being sent here. its not very readable.